image/spec/README.md

```diff
@@ -0,0 +1,47 @@
+# Docker Image Specification v1.
+
+This directory contains documents about Docker Image Specification v1.X.
+
+Distribution and archival specifications such as file layouts and manifests are no longer used
```

Comment:
This statement is odd. Distribution specifications are 100% in use in docker today.

Reply:
done